frontend/src/components/filtertable.js

```diff
@@ -26,11 +26,11 @@ import { Settings } from '../settings';
 import { HttpClient } from '../services/http';
 import { toAPIFilter } from '../utilities';
 
-import { TableEmptyState, TableErrorState } from './tablestates';
+import {TableEmptyState, TableErrorState} from './tablestates'
```

Comment:
@mshriver we should probably have some linter that will autofix all of these spaces and `;`

Reply:
correct - I included the prettier dependency in one of the modernization/eslint PRs, but haven't applied it.

Plan is to apply it following this branch merging to main